tested/main.py

```diff
@@ -30,10 +33,19 @@ def run(config: DodonaConfig, judge_output: IO):
     _, ext = os.path.splitext(config.test_suite)
     is_yaml = ext.lower() in (".yaml", ".yml")
     if is_yaml:
+        if language:
+            textual_suite, missing_keys = run_translation_with_str(
+                textual_suite,
+                language=language,
+                to_file=False,
```

Comment:
I think `false` is a sensible default for `to_file` for a method that starts with a string as input. (I am not even sure if we need `to_file` as an option at all for this method)